src/test/scala/calc/RunTest.scala

```diff
@@ -18,7 +19,7 @@ import org.scalajs.jsenv.JSConsole
  *  result.
  */
 class RunTest {
-
+  val r = Random
```

Comment:
Hmm, I don't think `if (1) let x = 1 in x*2 else x*3` is a valid statement. Because the scope of `x` is just the then phase of the conditional statement, not including else phase.